packages/data/README.md

```diff
@@ -508,6 +508,26 @@ _Returns_
 
 -   `Object`: Object containing the action creators.
 
+### experiments
+
+The experimental APIs exposed by the `@wordpress/data` package.
+Only available to core packages. These APIs are not stable and may
+change without notice. Do not use outside of core.
+
+_Usage_
+
+```js
+import { unlock } from '../experiments';
+import { experiments as dataExperiments } from '@wordpress/data';
+const { registerPrivateSelectors } = unlock( dataExperiments );
+
+import { store as blockEditorStore } from './store';
+import { __unstableSelectionHasUnmergeableBlock } from './store/selectors';
+registerPrivateSelectors( store, {
+	__experimentalHasContentRoleAttribute,
+} );
+```
```

Comment:
These selectors don't match, which adds confusion.

Reply:
ah snap good spot, I'll fix this in #46131

Reply:
https://github.com/WordPress/gutenberg/pull/46131/commits/8b5103cd63a26d5c4b8c18ec7df92f985fe3a9a4